### ISSUE TITLE
How do you apply weight normalization?

### ISSUE THREAD

**Issue by taki0112** [NickShahML/tensorflow_with_latest_papers#20]
How do you apply weight normalization?
For example,
```python
x = conv_layer(input)
x = weight_normalization(x)
```

Is this the right way to apply?

**Comment by taki0112** [comment]
or 
```python
tf.layers.conv2d(inputs=x, filters=filter, kernel_size=kernel, kernel_initializer=weight_norm(he_init), strides=stride, padding=padding)
```
Is this ?
